api/types/transfer/imagestore.proto

```diff
@@ -0,0 +1,36 @@
+/*
+	Copyright The containerd Authors.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+syntax = "proto3";
+
+package containerd.v1.types;
+
+option go_package = "github.com/containerd/containerd/api/types/transfer";
+
+message ImageStore {
+	string name = 1;
+	map<string, string> labels = 2;
+
+	// Content filters
+
+	repeated string platforms = 3; // Does this need a separate type? MatchComparer
+	bool all_metadata = 4;
+	uint32 manifest_limit = 5;
+
+	// Unpack Configuration
+
+	repeated string unpack_platforms = 6;
```

Comment:
Whats the difference between platforms and unpack_platforms?

Reply:
I should probably just change `unpack_platforms` into a separate type. What it is really indicating is different unpack configurations which may be done in parallel. Normally this would be keyed on platform and snapshotter would be inferred from the platform. The client may want control over that to target the platform and the snapshotter.